examples/react/store-context/src/index.tsx

```
@@ -0,0 +1,151 @@
+import ReactDOM from 'react-dom/client'
+import {
+  useAtom,
+  useCreateAtom,
+  createStoreContext,
```

ESLint

[error] Member 'createStoreContext' of the import declaration should be sorted alphabetically.

(sort-imports)